### PR TITLE
Add Lost Sword to Curse of Strahd

### PR DESCRIPTION
**Pull request by SteSaunders**
Add the Lost Sword found in Strahd's Castle in Curse of Strahd.

### CODE REVIEW

File: supplements/curse-of-strahd/items.xml [hunk 1]
```diff
@@ -161,4 +161,28 @@
...
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type" addition="shortsword">Weapon</set>
+			<set name="rarity">Rare</set>
+			<set name="enhancement">1</set>
+			<set name="weapon">Shortsword</set>
+			<set name="attunement">true</set>
+		</setters>
+		<rules>
+			<grant type="Spell" id="ID_PHB_SPELL_CRUSADERS_MANTLE" level="3" spellcasting="Paladin" prepared="true" />
```
Comment (NickVendel) [inline supplements/curse-of-strahd/items.xml]: You probably should remove `spellcasting="Paladin"` from the spell

Comment (SteSaunders) [reply supplements/curse-of-strahd/items.xml]: Done. Also fixed a type in the Description for the Sun Sword.

**Comment by swdriessen** [comment]
On what page can i find this item?

**Comment by NickVendel** [comment]
page 81
изображение

**Comment by swdriessen** [comment]
You can remove the `prepared="true"` and you can change the attunement setter to `<set name="attunement" addition="by a lawful good creature">true</set>`

I have not implemented the 'can't equip unless' requirements yet, so you'll need to put requirements on the rules for now. (`requirements="ID_ALIGNMENT_LAWFUL_GOOD"`)

**Comment by SteSaunders** [comment]
> You can remove the `prepared="true"` and you can change the attunement setter to `<set name="attunement" addition="by a lawful good creature">true</set>`
> 
> I have not implemented the 'can't equip unless' requirements yet, so you'll need to put requirements on the rules for now. (`requirements="ID_ALIGNMENT_LAWFUL_GOOD"`)

Done

**Comment by swdriessen** [comment]
Thanks

**Pull request merged**